scripts/cv_api_master.py

```diff
@@ -254,9 +254,102 @@
-    "title": "Models",
+FUNCTIONS_MASTER = {
+    "path": "functions/",
+    "title": "Functions",
+    "toc": True,
+    "children": [
+        {
+            "path": "efficientnetv2",
+            "title": "EfficientNetV2 models",
+            "generate": [
+                "keras_cv.models.EfficientNetV2B0",
+                "keras_cv.models.EfficientNetV2B1",
+                "keras_cv.models.EfficientNetV2B2",
+                "keras_cv.models.EfficientNetV2B3",
+                "keras_cv.models.EfficientNetV2S",
+                "keras_cv.models.EfficientNetV2M",
+                "keras_cv.models.EfficientNetV2L",
+            ],
+        },
+        {
+            "path": "densenet",
+            "title": "DenseNet models",
+            "generate": [
+                "keras_cv.models.DenseNet121",
+                "keras_cv.models.DenseNet169",
+                "keras_cv.models.DenseNet201",
+            ],
+        },
+    ]
+}
+
+BACKBONES_MASTER = {
+    "path": "backbones/",
+    "title": "Backbones",
+    "toc": True,
+    "children": [
+        {
+            "path": "resnet_v1",
+            "title": "ResNetV1 backbones",
+            "generate": [
+                "keras_cv.models.ResNetBackbone",
+                "keras_cv.models.ResNetBackbone.from_preset",
+                "keras_cv.models.ResNet18Backbone",
+                "keras_cv.models.ResNet34Backbone",
+                "keras_cv.models.ResNet50Backbone",
+                "keras_cv.models.ResNet101Backbone",
+                "keras_cv.models.ResNet152Backbone",
+            ],
+        },
+        {
+            "path": "resnet_v2",
+            "title": "ResNetV2 backbones",
+            "generate": [
+                "keras_cv.models.ResNetV2Backbone",
+                "keras_cv.models.ResNetV2Backbone.from_preset",
+                "keras_cv.models.ResNet18V2Backbone",
+                "keras_cv.models.ResNet34V2Backbone",
+                "keras_cv.models.ResNet50V2Backbone",
+                "keras_cv.models.ResNet101V2Backbone",
+                "keras_cv.models.ResNet152V2Backbone",
+            ],
+        },
+        {
+            "path": "csp_darknet",
+            "title": "CSPDarkNet backbones",
+            "generate": [
+                "keras_cv.models.CSPDarkNetBackbone",
+                "keras_cv.models.CSPDarkNetBackbone.from_preset",
+                "keras_cv.models.CSPDarkNetTinyBackbone",
+                "keras_cv.models.CSPDarkNetSBackbone",
+                "keras_cv.models.CSPDarkNetMBackbone",
+                "keras_cv.models.CSPDarkNetLBackbone",
+                "keras_cv.models.CSPDarkNetXLBackbone",
+            ],
+        },
+        {
+            "path": "yolov8",
+            "title": "YOLOV8 backbones",
+            "generate": [
+                "keras_cv.models.YOLOV8Backbone",
+                "keras_cv.models.YOLOV8Backbone.from_preset",
+            ],
+        },
+        {
+            "path": "mobilenetv3",
```

Comment:
Why not document the "large" and "small" aliases for consistency?

Reply:
Added.